src/commands/CommandManager.cpp

```diff
@@ -171,12 +171,13 @@ struct CommandListEntry
    bool useStrictFlags{ false };
 };
 
-NonKeystrokeInterceptingWindow::~NonKeystrokeInterceptingWindow()
-{
-}
+NonKeystrokeInterceptingWindow::~NonKeystrokeInterceptingWindow() = default;
 
-TopLevelKeystrokeHandlingWindow::~TopLevelKeystrokeHandlingWindow()
+TopLevelKeystrokeHandlingWindow::~TopLevelKeystrokeHandlingWindow() = default;
+
+bool TopLevelKeystrokeHandlingWindow::HandleCommandKeystrokes()
 {
+   return false;
```

Comment:
Should the default return true, to preserve old behavior except in cases where you override it?

Reply:
Right, thanks!